packages/docs/concepts/connections-and-requests.yaml

```diff
@@ -24,30 +24,28 @@ _ref:
         type: Markdown
         properties:
           content: |
-            ### TLDR
-              - `connections` define links to other services, like connecting to a database. They are defined at the root of the lowdefy configuration.
-              - `requests` use connections to make a call to the connected external services.
-              - Use the [`_secret`](/_secret) operator to reference API keys or other secrets as required - do not code secrets into your config or commit secrets to your config source control.
-
-            -----------
-            In a Lowdefy app you can integrate with other services like API's or databases using `connections` and `requests`. Connections configure the connection settings to the service, and often contain parameters like connection strings, urls and secrets like passwords or API keys. Requests are used to interact with the connection, like inserting a data record, executing a query or calling a API end-point.
+            In a Lowdefy app you can integrate with other services like API's or databases using `connections` and `requests`. Connections configure the connection settings to the service, and often contain parameters like connection strings, urls and secrets like passwords or API keys. Requests are used to interact with the connection, such as inserting a data record, executing a query or calling an API end-point.
 
-      - id: alert2
+      - id: secrets_alert
         type: Alert
         properties:
           type: warning
           showIcon: false
-          message: Sensitive information like passwords or API keys are often required to use external services. The _secret operator should be used to reference these secrets, they should never be coded directly in your app, or committed to source control.
+          message: Sensitive information like passwords or API keys are often required to use external services. The <a href = "./_secret">_secret</a> operator should be used to reference these secrets, they should never be coded directly in your app, or committed to source control.
 
       - id: md2
         type: Markdown
         properties:
           content: |
-            # Connections
+            To implement requests, the following steps are required:
+            - define a connection.
```

Comment:
Should these start with capitals?

Reply:
I think they should. 

Change made in 97d1d764505e2c23bfcbf67116271b2ab8a853f0